examples/timeseries/timeseries_forecasting_with_lstm_for_temperature_prediction.py

```diff
@@ -0,0 +1,184 @@
+"""
+Title: Timeseries forecasting with LSTM for temperature prediction
+Author: [Falak Shah](https://github.com/falaktheoptimist)
+Date created: 2020/06/28
+Last modified: 2020/06/28
+Description: Predict average next day temperature using LSTM based model.
+"""
+
+"""
+## Introduction
+This script demonstrates how you can use an LSTM model for time series
+prediction. Usage of timeseries_dataset_from_array is explained for
+preparing input data for the model.
+"""
+
+"""
+## Setup
+As of writing this tutorial, timeseries_dataset_from_array is available only
+in TF-nightly.
+"""
+
+# tf_nightly needed for timeseries_dataset_from_array
+#! pip uninstall -y tensorflow
+#! pip install tf-nightly
```

Comment:
You can remove these lines. You just need to say "this example requires TensorFlow 2.3 or higher"

Reply:
Done